components/homme/CMakeLists.txt

```diff
@@ -50,8 +50,16 @@ IF (HOMME_BUILD_EXECS OR BUILD_HOMME_TOOL)
 ELSE ()
   SET(BUILD_HOMME_WITHOUT_PIOLIBRARY ON CACHE BOOL "")
 ENDIF ()
+
+SET(HOMME_BUILD_SCORPIO ON)
 IF(BUILD_HOMME_WITHOUT_PIOLIBRARY)
   MESSAGE(STATUS "This configuration builds without PIO library and NetCDF calls")
+  SET(HOMME_BUILD_SCORPIO OFF)
+ELSE()
+  IF (TARGET pioc)
```

Comment:
Shouldn't Homme check for `piof` rather than pioc? Unless you use the C interfaces, that's the lib you need to link, no?

Reply:
Possibly. But note this isn't really imporant. The main thing, as you pointed out, is we can't have Homme buildings its own SCORPIO if it's already on the link line. `pioc` seems the safer thing to check. `piof` would imply that somehow we're robust to building `piof` in Homme and `pioc` elsewhere, which we're not.